### PR TITLE
Fixes params in /gtfo

### PR DESCRIPTION
**Pull request by LegendIsAwesomes**
Fixes the gtfo params bit in: https://github.com/TotalFreedom/TotalFreedomMod/issues/848 pretty easy fix IMO just some editing.

### CODE REVIEW

File: src/me/StevenLawson/TotalFreedomMod/Commands/Command_gtfo.java [hunk 1]
```diff
@@ -16,7 +16,7 @@
 import org.bukkit.entity.Player;
 
 @CommandPermissions(level = AdminLevel.SUPER, source = SourceType.BOTH, blockHostConsole = true)
-@CommandParameters(description = "Makes someone GTFO (deop and ip ban by username).", usage = "/<command> <partialname>")
+@CommandParameters(description = "Makes someone GTFO (deop and ip ban by username).", usage = "/<command> <partialname> <reason>")
```
Comment (JeromSar) [inline src/me/StevenLawson/TotalFreedomMod/Commands/Command_gtfo.java]: The reason is an optional parameter. This is the incorrect format.

**Comment by JeromSar** [comment]
Please implement all your changes from the referred PRs (see above).

**Comment by LegendIsAwesomes** [comment]
@JeromSar The new contributing guidelines say that each issue should have a seperate PR, So will this change

**Comment by Wild1145** [comment]
@LegendIsAwesomes Currently issue #848 is the single issue for this, however this issue archives multiple tasks, this PR would need to achieve all of the designated tasks for it to be able to be a full resolution and ready to merge.

**Comment by LegendIsAwesomes** [comment]
Okay, would I be able to create a new PR since this wasn't  done in the github app.

**Comment by JeromSar** [comment]
@LegendIsAwesomes @Wild1145 This PR does not actually concern the issue in question, but rather it fixes the invalid argument description /gtfo had. Therefore, I'm going to make an _exception_ and accept this PR. Normally, a new issue would have to be made.

**Comment by LegendIsAwesomes** [comment]
@JeromSar why was the ready to merge removed.

**Comment by DUFCLiam** [comment]
Needed CLA checked, Should be ready to merge now.

**Comment by LegendIsAwesomes** [comment]
Needs openin'

**Comment by JeromSar** [comment]
@LegendIsAwesomes This PR can not be re-opened. The original repository has been deleted.

**Comment by DUFCLiam** [comment]
I will create new PR for this then. @JeromSar (Copy code exactly)

**Comment by LegendIsAwesomes** [comment]
No.. This is my code I will create it

**Pull request closed without merging**